microservices-orders/docs/index.rst

```diff
@@ -138,10 +138,18 @@ Local:
 
 Docker:
 
-* Docker version 17.06.1-ce
-* Docker Compose version 1.14.0 with Docker Compose file format 2.1
+* Docker version >= 19.00.0
+* Docker Compose version >= 1.25.0 with Docker Compose file format 3
 * In Docker's advanced `settings <https://docs.docker.com/docker-for-mac/#advanced>`__, increase the memory dedicated to Docker to at least 8GB (default is 2GB)
 
+|ccloud|:
+
+* Docker version >= 19.00.0
+* Docker Compose version >= 1.25.0 with Docker Compose file format 3
+* In Docker's advanced `settings <https://docs.docker.com/docker-for-mac/#advanced>`__, increase the memory dedicated to Docker to at least 8GB (default is 2GB)
+* |ccloud| account. The `Confluent Cloud <https://www.confluent.io/confluent-cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.microservices-orders?>`__ home page can help you get setup with your own account if you do not yet have access.
+* |ccloud| CLI. See :ref:`Install and Configure the Confluent Cloud CLI <cloud-cli-install>`.
+
```

Comment:
@rspurgeon please decide on best placement for this, but include this please https://github.com/confluentinc/examples/blob/5.5.1-post/ccloud/docs/includes/ccloud-promo-code.rst (you can reference it as an `include` directive)

Reply:
Added